### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by xxm-wu**
add cairo vm and other libraries

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,9 +36,20 @@ Other Starknet/Cairo related lists: [Other lists](#other-lists)
 - [`erc721.cairo`](https://github.com/reddio-com/cairo/blob/main/token/ERC721/erc721.cairo) – An early ERC721 implementation
 - [`suna`](https://github.com/auditless/suna) – Typesafe primitives for Cairo 1.0 with a focus on DeFi protocols
 - [`arcade-accounts`](https://github.com/BibliothecaDAO/arcade-account) - Starknet Accounts with fine grain permissions for use in onchain games
+- [`graffiti`](https://github.com/milancermak/graffiti/) - A Cairo library for building XML based documents
```
Comment (Pet3ris) [inline README.md:39]: duplicate

File: README.md [hunk 2]
```diff
@@ -35,9 +36,20 @@ Other Starknet/Cairo related lists: [Other lists](#other-lists)
...
+- [`graffiti`](https://github.com/milancermak/graffiti/) - A Cairo library for building XML based documents
 
 **[back to top](#contents)**
 
+## Cairo VM
```
Comment (Pet3ris) [inline README.md:43]: nit: should be a newline after this

**Review by Pet3ris — CHANGES_REQUESTED** [review]
Thank you for your contribution, just small nits

**Review by xxm-wu — COMMENTED** [review]
resolved

**Review by Pet3ris — APPROVED** [review]
LGTM

**Pull request merged**